This makes wget work again in case /etc/hosts is missing.

diff -Naur glibc-2.11/nss/nss_files/files-hosts.c glibc-2.11/nss/nss_files/files-hosts.c
--- glibc-2.11/nss/nss_files/files-hosts.c	2009-10-30 18:17:08.000000000 +0100
+++ glibc-2.11/nss/nss_files/files-hosts.c	2009-12-18 22:37:06.642163627 +0100
@@ -426,7 +426,7 @@
   else
     {
       *errnop = errno;
-      *herrnop = NO_DATA;
+      *herrnop = HOST_NOT_FOUND;
     }
 
   __libc_lock_unlock (lock);
